core/src/FieldTypeDate/FieldTypeDate.js

```diff
@@ -13,8 +13,13 @@ import { FieldLabel } from "../FieldLabel";
 import styles from "./FieldTypeDate.less";
 export const FieldTypeDate = React.memo(function FieldTypeDate(props) {
   // console.log("FieldTypeDate:render");
+  const ref = useRef();
 
-  const onChange = date => {
+  const openCalendar = () => {
+    setTimeout(() => ref.current.flatpickr.open(), 0);
```

Comment:
why is this wrapped in a `setTimeout`?

Reply:
@d88naimi would you add a comment which includes a reference to the linked issue and note why it's necessary. Want to make sure we remember this in the future.